packages/storage-plus/src/iter_helpers.rs

```diff
@@ -1,5 +1,3 @@
-#![cfg(feature = "iterator")]
```

Comment:
I think this was here for "no dead code". But if it compiles, nice to remove it

Reply:
Oh, this was a mistake. Will see the impact.